src/Camelotia.Presentation/DesignTime/DesignTimeFolderViewModel.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Camelotia.Presentation.Interfaces;
+using System.Collections.Generic;
+
+namespace Camelotia.Presentation.DesignTime
+{
+    public class DesignTimeFolderViewModel : IFolderViewModel
+    {
+        public string Name { get; set; }
+        public IEnumerable<string> Children { get; set; }
```

Comment:
Nit: worth inserting a line break after `Name { get; set; }` to be consistent with formatting in other files. Planning to bring in analyzers similar to `stylecop`, and probably autoformatting tools in the future to address such minor issues.